app/cypress/integration/cif/contacts.spec.js

```diff
@@ -48,4 +48,43 @@ describe("The contacts page", () => {
     cy.contains("email already exists").should("be.visible");
     cy.contains(/please enter a value/i).should("be.visible");
   });
+
+  it("Validates email", () => {
+    cy.visit("/cif/contacts");
+    cy.get("h2").contains("Contacts");
+    cy.get("button").contains("Add").click();
+
+    cy.get("input[aria-label='Given Name']").should("be.visible").type("Bob");
+    cy.get("input[aria-label='Family Name']").type("Loblaw");
+    cy.get("input[aria-label=Email]").type("bob.loblaw.ca");
+    cy.get("input[aria-label='Company Name']").type("ABC");
+
+    cy.contains("Changes saved").should("be.visible");
+    cy.get("input[aria-label=Phone]").type("1234567890");
+    cy.contains("Changes saved").should("be.visible");
+    cy.get("button").contains("Submit").click();
+    cy.contains("Please enter in the format: name@example.com").should("be.visible");
+    
+  });
+
+  it("Validates phone number", () => {
+    cy.visit("/cif/contacts");
+    cy.get("h2").contains("Contacts");
+    cy.get("button").contains("Add").click();
+
+    cy.get("input[aria-label='Given Name']").should("be.visible").type("Bob");
+    cy.get("input[aria-label='Family Name']").type("Loblaw");
+    cy.get("input[aria-label=Email]").type("bob@loblaw.ca");
+    cy.get("input[aria-label='Company Name']").type("ABC");
+
+    cy.contains("Changes saved").should("be.visible");
+    cy.get("input[aria-label=Phone]").type("12345");
+    cy.contains("Changes saved").should("be.visible");
+
+    cy.get("button").contains("Submit").click();
+    cy.contains("Please enter in a valid phone number format (e.g. 123 456 7890)").should("be.visible");
+  });
```

Comment:
We should roll these 2 test cases into the "Shows correct validation errors" one just above, it'll speed up the test quite a bit - and remove a lot of repeated testing code